swagger.js

```diff
@@ -32,6 +32,86 @@ const doc = {
         },
       },
     },
+    definitions: {
+      Order: {
+        $_id: 'idabc123456',
+        $user: {
+          $id: 'idabc123456',
+          $name: 'John Doe',
+          $username: 'johndoe',
+          $email: 'john@example.com',
+          $role: { '@enum': ['user', 'admin'] },
+        },
+        $orderStatus: {
+          '@enum': [
+            'pending',
+            'payment_successful',
+            'payment_failed',
+            'cancelled',
+            'complete',
+          ],
+          default: 'pending',
+        },
+        orderItems: [
+          {
+            $album: '5f7f1f7a5c3f2b2d9c1b0b9d',
+            $quantity: 2,
+            $_id: 'idabc12345',
+          },
+        ],
+        $subtotal: 9.99,
+        $tax: 0.75,
+        $total: 10.74,
+        $createdAt: '2023-08-08T20:45:00.942Z',
+        $updatedAt: '2023-08-08T20:45:00.942Z',
+        paymentIntentId: 'pi_abc123456',
+      },
+      NewOrder: {
+        type: 'object',
+        properties: {
+          album: {
+            type: 'string',
+            example: 'Album ID or Name',
+          },
+          quantity: {
+            type: 'integer',
+            example: 2,
+          },
+        },
+      },
+    },
```

Comment:
In swagger-autogen, by default we are expected to describe objects/responses/requests as their literal form / or an example of what they would look like, rather than using raw Swagger/OpenAPI notation (though swagger-autogen does have an option for providing raw syntax in some cases where needed).

So right now, this block is saying that when creating a new order, we have to pass in fields like "type" and "properties" which is incorrect:

Rather, we should be writing this as exactly how we expect the frontend or any other API client to pass information to this endpoint:

```suggestion
      NewOrder: {
        $subtotal: 100,
        $tax: 0.15,
        $total: 115,
        $orderItems: [
          { album: "abcdef12345", quantity: 2 }
        ]
      },
    },
```